build: new provider line notify (#2049)
novuhq/novu: providers/linenotify/README.md

> # Novu Linenotify Provider

Reviewer: Please rename the folder of the provider to `line-notify` instead of `linenotify`.

> A Linenotify chat provider library for [@novu/node](https://github.com/novuhq/novu)

Reviewer: ```suggestion
A LINE Notify chat provider library for [@novu/node](https://github.com/novuhq/novu)
```

## Usage

```javascript

> const { LinenotifyProvider } = require('@novu/linenotify-provider');

Reviewer: ```suggestion
const { LineNotifyProvider } = require('@novu/line-notify');
```

> const provider = new LinenotifyProvider({

Reviewer: ```suggestion
const provider = new LineNotifyProvider({
```
